scripts/find_packages.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+import os.path
+import sys
+
+scripts_path = os.path.dirname(os.path.realpath(__file__))
+bb_lib_path = os.path.abspath(scripts_path + '/../../poky/bitbake/lib')
+sys.path = sys.path + [bb_lib_path]
+
+import bb.fetch2
+import bb.tinfoil
+
+
+def print_deps(tinfoil, abcd_file, rn):
+    try:
+        info = tinfoil.get_recipe_info(rn)
+    except:
```

Comment:
Bare except are discouraged (and marked as warnings by flake8): https://www.python.org/dev/peps/pep-0008/#programming-recommendations. It can usually be replaced by `except Exception:` which, at least, doesn't catch KeyboardInterrupt and plays more nicely with python's convention with exceptions.